src/Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolValidatorTest.cs

```diff
@@ -80,55 +46,14 @@ public async void CheckFinalTransaction_WithNoLockTimeAndValidTxTime_ReturnsTrue
 
             // Do not set an nLockTime for this test.
 
-            // Set the transaction time to a recent value.
-            tx.Time = context.SrcTxs.Last().Time + 100;
-
-            tx.Sign(network, minerSecret, false);
-
-            var state = new MempoolValidationState(false);
-
-            bool isSuccess = await validator.AcceptToMemoryPool(state, tx);
-            Assert.True(isSuccess, "Transaction with no nLockTime and a valid transaction time should have been accepted.");
-        }
-
-        [Fact(Skip = "Awaiting fix for issue #2474")]
-        public async void CheckFinalTransaction_WithStandardLockTimeAndExpiredTxTime_FailsAsync()
```

Comment:
Why delete it though?

Reply:
I'm not sure what an 'expired' transaction time is meant to be; if there is a necessary test case here that's missing then I can implement it